src/mapper.proto

```diff
@@ -1,24 +1,42 @@
 syntax = "proto3";
 package helium;
 
-message mapper {
-  oneof message {
-    mapper_gps gps = 1;
-    mapper_scan scan = 2;
-    mapper_attach attach = 3;
-  }
+service ScanUpload {
+  rpc Upload(mapper_msg) returns (scan_upload_reply) {}
+}
+
+message scan_upload_reply { scan_upload_status status = 1; }
+enum scan_upload_status {
+  SUCCESS = 0;
```

Comment:
same comment here about casing. I would assume the code generator would do correct thing for these?

Reply:
I believe it's a convention to have then enums in `CAPITALS_WITH_UNDERSCORES`? agreed about the other one though